renderdoc/driver/vulkan/vk_msaa_array_conv.cpp

```diff
@@ -337,6 +332,10 @@ void VulkanDebugManager::CopyDepthTex2DMSToArray(VkImage destArray, VkImage srcM
   {
     rpbegin.framebuffer = fb[i];
 
+    VkCommandBuffer cmd = m_pDriver->GetNextCmd();
+
+    ObjDisp(cmd)->BeginCommandBuffer(Unwrap(cmd), &beginInfo);
```

Comment:
What's the reasoning behind moving this command buffer to be per-layer instead of a single command buffer for all copies?